defaults/krill-pubd.conf

```diff
@@ -122,6 +122,59 @@
 ### post_limit_rfc8181 = 33554432
 
 
+######################################################################################
+#                                                                                    #
+#                            REPOSITORY RRDP SETTINGS                                #
+#                                                                                    #
+######################################################################################
+
+
+#
+# RRDP DELTA FILES
+#
+# The RRDP protocol uses deltas to allow Relying Parties to synchronise efficiently.
+# If Relying Parties cannot retrieve deltas, they will fall back to using snapshots.
+# The RRDP protocol specifies that no more deltas should be presented than the total
+# size of the snapshot. However, it is good to limit the amount of deltas further than
+# this because:
+#  1) big notification files are expensive to serve and parse
+#  2) it's not just size, because of latency parsing many small deltas may
+#     be more time consuming than falling back to a delta
+#
+# So, the objective would be to optimize things such that the notification.xml files
+# remain relatively small, while still allowing the vast majority of Relying Party
+# implementations to use deltas, under normal operations.
+#
+# We have use the following strategy to prune excessive deltas:
+#
+#  - never keep more than the size of the snapshot
+#  - always keep 'retention_delta_files_min_nr' files
+#  - always keep 'retention_delta_files_min_seconds' files
+#  - beyond this:
+#    - never keep more than 'retention_delta_files_max_nr'
+#    - never keep older than 'retention_delta_files_max_seconds'
+#    - keep the others
+#
+# With these defaults:
+# retention_delta_files_min_nr = 5
+# dflt_retention_delta_files_min_seconds = 1200 (20 mins: twice the interval of commonly used RPs)
+# dflt_retention_delta_files_max_seconds = 7200 (2 hours: twice the interval of slowest RPs)
+# dflt_retention_delta_files_max_nr = 50
+
+# Time to keep any files still referenced by notification files updated up to X seconds ago. We
+# should not delete these files too eagerly or we would risk that RPs with an old notification file
+# try to retrieve them, without success.
+#
+# retention_old_notification_files_seconds = 600 (10 mins)
```

Comment:
Does the value format of this setting support "(10 mins)" - I assume not. I prefer it when commented out settings just work if the leading comment character is removed.

Reply:
indeed, removed